backend/apps/accounts/views.py

```
@@ -600,625 +120,46 @@ def post(self, request):
-
-        # Progress stats
-        total_score = (
-            LessonProgress.objects.filter(user=user).aggregate(total=Sum("score"))[
-                "total"
-            ]
-            or 0
-        )
-        completed_lessons = LessonProgress.objects.filter(
-            user=user, completed=True
-        ).count()
+        except PasswordResetToken.DoesNotExist:
```

Ruff (0.15.20)

[error] Undefined name `PasswordResetToken`

(F821)